src/scripts/farming/Farming.ts

```diff
@@ -1027,6 +1031,12 @@ class Farming implements Feature {
             .reduce((acc, p) => acc * (p.emittingAura.value() ?? 1), 1);
     }
 
+    addPlotAuras(auraType: AuraType): number {
+        return this.plotList
+            .filter(p => p.emittingAura.type() === auraType)
+            .reduce((acc, p) => acc + (p.emittingAura.value() ?? 1), 0);
```

Comment:
```suggestion
            .reduce((acc, p) => acc + (p.emittingAura.value() ?? 0), 0);
```
This should probably default to 0.